src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Represents a list of Tasks.
+ *
+ * @author Joshua Yong
+ */
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+
+    /**
+     * Class constructor.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Overloaded class constructor which initializes the TaskList with existing tasks.
+     *
+     * @param tasks An ArrayList of tasks to be stored in the TaskList.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public ArrayList<Task> getTasks() {
```

Comment:
Perhaps you can consider modifying the names of one of the methods (`getTasks()` or `getTask(int i)`) so that they are more distinguishable?